### PR TITLE
Update office new moves queue heading: 165749867

### PR DESCRIPTION
**Pull request by lynzt**
## Description

Update office `New Moves` heading to `New Moves/Shipments` and update related tests.
Update the link on a moves page from `New Moves Queue` to `New Moves/Shipments Queue`.

## Setup

`make server_run`
`make office_client_run`

## Code Review Verification Steps

* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165749867) for this change

## Screenshots

Screen Shot 2019-05-24 at 7 51 26 AM

Screen Shot 2019-05-29 at 8 18 30 AM

### CODE REVIEW

File: cypress/integration/office/officeCSRFProtect.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ describe('testing CSRF protection', function() {
 
   it('tests dev login with both unmasked and masked token', function() {
     cy.signInAsUserPostRequest(officeAppName, userId);
-    cy.contains('Queue: New Moves');
```
Comment (donaldthai) [inline cypress/integration/office/officeCSRFProtect.js]: I know the pivotal story said "New Moves/Shipments" but I'm wondering if we should change the case to sentence case? Eg: "New moves/shipments"

File: cypress/integration/office/officeCSRFProtect.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('testing CSRF protection', function() {
...
+    cy.contains('New Moves/Shipments');
```
Comment (donaldthai) [inline cypress/integration/office/officeCSRFProtect.js]: I know the pivotal story said "New Moves/Shipments" but I'm wondering if we should change the case to sentence case? Eg: "New moves/shipments"

Comment (kahlouie) [reply cypress/integration/office/officeCSRFProtect.js]: Agree, especially if it says `HHG shipments` below.

Comment (lynzt) [reply cypress/integration/office/officeCSRFProtect.js]: @donaldthai / @kahlouie - design gave an ok for the capitalization - do we want to push back and get a 2nd opinion?https://defensedigitalservice.slack.com/archives/G8MKQ1AJF/p1559221253000900?thread_ts=1558703910.001500&cid=G8MKQ1AJF

**Comment by lynzt** [comment]
@garrettqmartin8 design said the line break for the queue name was ok: https://defensedigitalservice.slack.com/archives/G8MKQ1AJF/p1559221360001100?thread_ts=1558703910.001500&cid=G8MKQ1AJF

File: cypress/integration/office/officeCSRFProtect.js (hunk 2 above)
Comment (donaldthai) [reply cypress/integration/office/officeCSRFProtect.js]: If design said this was okay then I'm good with it

**Pull request merged**